Utils/HandlerUtil.py

```diff
@@ -81,14 +81,14 @@ def __init__(self,name):
         self._seq_no = -1
         self._status_file = None
         self._settings_file = None
-        self._config = None
+        self._config = None        
         return
 
 class HandlerUtility:
-    def __init__(self, log, error, s_name=None, l_name=None, extension_version=None):
+    def __init__(self, log, error, s_name=None, l_name=None, extension_version=None, fromTelemetry = False):
```

Comment:
Is this change really for all extensions, or just for OMS?  If it is just OMS, I wouldn't bother with anything else.